client/app/_services/app-health/app-health.service.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const appHealthService = function appHealthService(
+    $http,
+    API_URL,
+) {
+    'ngInject';
+
+    let status = false;
+
+    function checkServerStatus() {
+        return $http.get(`${API_URL}/api/status`, { transformResponse: [data => data] });
+    }
+
+    return {
+        checkServerStatus,
+
+        get isHealthy() { return status; },
+        set isHealthy(newStatus) { return status = newStatus; },
```

Comment:
maybe we can use function `changeHelthyStatus(boolean)` instead of setter. It's more clear for me in other parts of the code

Reply:
fixed